File: current/source/includes/indexes/index-starter-code.scala
import org.mongodb.scala._
import org.mongodb.scala.model.SearchIndexModel

import java.util.concurrent.TimeUnit
import scala.concurrent.Await
import scala.concurrent.duration.Duration

import org.mongodb.scala.model.Indexes

object SearchIndexes {

  def main(args: Array[String]): Unit = {

    // Create a new client and connect to the server
    val mongoClient = MongoClient("<connection string URI>")

    val database = mongoClient.getDatabase("<database name>")
    val collection = database.getCollection("<collection name>")

    // Start example code here

    
    // End example code here

    Thread.sleep(1000)
    mongoClient.close()
  }
}

File: content/ruby-driver/v2.21/source/includes/indexes/index-starter-code.rb
require 'mongo'

# Replace the placeholders with your credentials
uri = "<connection string>"

# Sets the server_api field of the options object to Stable API version 1
options = { server_api: { version: "1" }}

# Creates a new client and connect to the server
client = Mongo::Client.new(uri, options)

database = client.use('<database name>')
collection = database[:<collection name>]

# Start example code here
    
# End example code here

client.close
